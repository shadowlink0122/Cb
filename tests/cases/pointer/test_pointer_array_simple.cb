// シンプルなポインタ配列テスト

void main() {
    println("=== Test: Simple pointer array ===");
    int a = 10, b = 20, c = 30;
    int*[3] ptrs = [&a, &b, &c];
    
    println(*ptrs[0]);  // 10
    println(*ptrs[1]);  // 20
    println(*ptrs[2]);  // 30
    
    println("=== Test passed ===");
}
