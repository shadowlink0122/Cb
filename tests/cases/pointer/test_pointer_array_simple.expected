=== Test: Simple pointer array ===
10
20
30
=== Test passed ===
